waltid-services/waltid-verifier-api/src/main/kotlin/id/walt/verifier/oidc/VerificationUseCase.kt

```diff
@@ -126,7 +127,7 @@ class VerificationUseCase(
             else -> TokenResponse.fromHttpParameters(tokenResponseParameters)
         }
         val sessionVerificationInfo = OIDCVerifierService.sessionVerificationInfos[session.id] ?: return Result.failure(
-            IllegalStateException("No session verification information found for session id!")
+            NotFoundException("No session verification information found for session id!")
```

Comment:
Is "session id" == "state parameter" ?

If so, what it we standardize the vocabulary to avoid confusion?